src/Jackett.Common/Definitions/noname-club.yml

```diff
@@ -7,6 +7,7 @@ type: public
 encoding: windows-1251
 links:
   - https://nnmclub.to/
+  - http://nnmclub5toro7u65.onion/ # will be https when there will be brave enough CA to issue certificate
```

Comment:
Change these all to *.onion.ly

Also, not sure the comments are necessary.

Reply:
- It seemed to me, looking through source code, that any non-https link should be commented with reason why it is non-https.
- Why use `onion.ly`? The point of PR was to provide single-click transition to Tor hidden sites, not tor2web.

Confirm change?

Reply:
`if there is a TOR domain for this, then appending .ly to the *.onion domain is an accepted way to get access.
We use this method for several indexers already.`
https://github.com/Jackett/Jackett/issues/6081#issuecomment-810031982

Vast majority of users won't be using Tor, so this supports them. As for Tor users, they just have to click and delete 3 characters.